install_config/router/f5_router.adoc

```diff
@@ -343,4 +345,9 @@ $ oc adm router \
     --external-host-vxlan-gw=10.131.0.5/14
 ----
 +
+[NOTE]
++
+====
+The `external-host-username` is a *F5 BIG-IP* user account with access to the Advanced Shell (Bash) on the F5 BIG-IP system.
+
```

Comment:
The markup above is a little off, and the sentence below has been pulled into the note box. line 352 should have another set of =s and perhaps no line space between the NOTE heading and the text. See:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions

Reply:
@bfallonf Fixed. Thanks!